src/SFTPClient.cpp

```diff
@@ -191,7 +191,6 @@ int SFTPClient::sftpConnected() {
 }
 
 int SFTPClient::disconnectUnlocked(bool force, int timeout_ms, AbstractDisconnectionHelper* adh, ExceptionSink* xsink) {
-   //printd(5, "SFTPClient::disconnectUnlocked() force: %d timeout_ms: %d adh: %p xsink: %p\n", force, timeout_ms, adh, xsink);
```

Comment:
why remove this line?  it's there so it can be uncommented for debugging purposes if needed in the future

Reply:
oops, I shouldn't have